go-runtime/ftl/ftltest/ftltest.go

```diff
@@ -497,3 +498,56 @@ func WaitForSubscriptionsToComplete(ctx context.Context) {
 	fftl := internal.FromContext(ctx).(*fakeFTL) //nolint:forcetypeassert
 	fftl.pubSub.waitForSubscriptionsToComplete(ctx)
 }
+
+// Call a Verb inline, applying resources and test behavior.
+func Call[VerbClient, Req, Resp any](ctx context.Context, req Req) (Resp, error) {
+	return call[VerbClient, Req, Resp](ctx, req)
+}
+
+// CallSource calls a Source inline, applying resources and test behavior.
+func CallSource[VerbClient, Resp any](ctx context.Context) (Resp, error) {
+	return call[VerbClient, ftl.Unit, Resp](ctx, ftl.Unit{})
+}
+
+// CallSink calls a Sink inline, applying resources and test behavior.
+func CallSink[VerbClient, Req any](ctx context.Context, req Req) error {
+	_, err := call[VerbClient, Req, ftl.Unit](ctx, req)
+	return err
+}
+
+// CallEmpty calls an Empty inline, applying resources and test behavior.
+func CallEmpty[VerbClient any](ctx context.Context) error {
+	_, err := call[VerbClient, ftl.Unit, ftl.Unit](ctx, ftl.Unit{})
+	return err
+}
+
+func call[VerbClient, Req, Resp any](ctx context.Context, req Req) (resp Resp, err error) {
+	ref := reflection.ClientRef[VerbClient]()
+	inline := server.Call[Req, Resp](ref)
+	moduleCtx := modulecontext.FromContext(ctx).CurrentContext()
+	override, err := moduleCtx.BehaviorForVerb(schema.Ref{Module: ref.Module, Name: ref.Name})
+	if err != nil {
+		return resp, fmt.Errorf("%s: %w", ref, err)
```

Comment:
nit: should we make this `"test harness failed to retrieve behavior for verb %s: %w"` and similar make L535 something like `"test harness failed to call verb %s: %w"`?

Reply:
good idea!